source/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,181 @@
+   Make sure you don’t select the drive with your Windows installation.
+
+#. Go through remaining steps to install |CL|.
+
+#. Reboot.
+
+#. During the BIOS POST stage, press the proper F-key for your system to
+   bring up the “Boot Menu”.
+
+#. Now, select the Windows partition (“Windows Boot Manager”) to
+   boot Windows or select the other one to boot |CL|.
+
+Advanced (Optional)
+*******************
+
+Boot Windows 10 OS through |CL| systemd-boot menu. If you prefer not to use
+your BIOS to load the boot menu and select an OS to boot, you can make |CL|
+the default bootloader and also add Windows 10 OS to its boot list.
+
+This option is also a workaround for BIOSes that don’t support booting more
+than one partition. Below shows how to add Windows to the |CL| systemd-boot
+menu and make it the default bootloader.
+
+.. TODO: restart here.
+
+1. Boot up the Clear Linux installer
+2. Open a Terminal window
+3. List the partitions to identify the Windows and Clear Linux EFI system partitions
+$ lsblk
+
+Example output:
+
+clrlinux@clr-live~ $ lsblk
+NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+loop0    7:0    0   2.3G  1 loop 
+sda      8:0    0 335.4G  0 disk 
+├─sda1   8:1    0   450M  0 part 
+├─sda2   8:2    0   100M  0 part 
+├─sda3   8:3    0    16M  0 part 
+├─sda4   8:4    0   286G  0 part 
+├─sda5   8:5    0   143M  0 part 
+├─sda6   8:6    0   244M  0 part 
+└─sda7   8:7    0  48.5G  0 part 
+sdb      8:16   1     7G  0 disk 
+├─sdb1   8:17   1   2.5G  0 part 
+└─sdb2   8:18   1   100M  0 part
+
+/dev/sda2 is the EFI system partition created by Windows
+/dev/sda4 is the primary Windows partition
+/dev/sda5 is the EFI system partition created by Clear Linux
+/dev/sda7 is the Clear Linux root partition
+
+4. Create mount points
+$ sudo mkdir /mnt/windows-efi
+$ sudo mkdir /mnt/clearlinux
+
+5. Mount the EFI system partition for Windows
+$ sudo mount /dev/sda2 /mnt/windows-efi
+
+6. Mount the Clear Linux root partition and its EFI system partition
+$ sudo mount /dev/sda7 /mnt/clearlinux
+$ sudo mount /dev/sda5 /mnt/clearlinux/boot
+
+7. Copy Windows’ bootloader and other data needed to boot Windows to the the Clear Linux EFI system partition
+$ sudo cp -r /mnt/windows-efi/EFI/Microsoft/ /mnt/clearlinux/boot/EFI/
+
+8. Make Clear Linux systemd-boot the default bootloader and add Windows Boot Manager 
+$ sudo bootctl install --esp-path=/mnt/clearlinux/boot
+
+9. Add a timeout value to the Clear Linux bootloader (systemd-boot) so that it will present the menu of bootable OSes and give you time to select the one you want to boot, otherwise, it will just boot Clear Linux right away
+$ sudo clr-boot-manager set-timeout 20
+$ sudo clr-boot-manager update
+
+10. Umount all partitions
+$ sudo umount /mnt/windows-efi /mnt/clearlinux/boot /mnt/clearlinux
+
+11. Reboot
+$ sudo reboot
+
+12. Remove Clear Linux installer USB thumb drive
+
+13. You should be presented with the systemd-boot menu like below
```

Comment:
Your Step 13 references a `systemd-boot` menu, but it doesn't show one. Want to provide one?

Reply:
I don't have a pre-OS screen capture device.  Do you have one?  I can do it if I can borrow one.

Reply:
I'll bring you one tomorrow.